src/components/pages/PageCardsEdit/PageCardsEdit.test.jsx

```diff
@@ -0,0 +1,296 @@
+
+        expect(cardSideFour.prop('value')).toBe(sideFour);
+      });
+    });
+
+    describe('Update props onChange to reflect new value', () => {
+      it('should display the changed card title', async () => {
+        const title = 'Changes';
+        const card = cardMock();
+
+        const { wrapper } = await setup({ card });
+        wrapper.update();
+        wrapper.find('input').at(0).simulate('change', { target: { value: title } });
+
+        expect(wrapper.find('input').at(0).prop('value')).toBe(title);
+      });
+
+      it('should display the changed card type', async () => {
+        const type = 'Circumstance';
+        const card = cardMock();
+
+        const { wrapper } = await setup({ card });
+        wrapper.update();
+        wrapper.find('select').at(0).simulate('change', { target: { value: type } });
+
+        expect(wrapper.find('select').at(0).prop('value')).toBe(type);
+      });
+
+      it('should display the changed value of card side one', async () => {
+        const sideOne = 'Career shift';
+        const card = cardMock();
+
+        const { wrapper } = await setup({ card });
+        wrapper.update();
+        wrapper.find('input').at(1).simulate('change', { target: { value: sideOne } });
+
+        expect(wrapper.find('input').at(1).prop('value')).toBe(sideOne);
+      });
+
+      it('should display the changed value of card side two', async () => {
+        const sideTwo = 'Move across country';
+        const card = cardMock();
+
+        const { wrapper } = await setup({ card });
+        wrapper.update();
+        wrapper.find('input').at(2).simulate('change', { target: { value: sideTwo } });
+
+        expect(wrapper.find('input').at(2).prop('value')).toBe(sideTwo);
+      });
+
+      it('should display the changed value of card side three', async () => {
+        const sideThree = 'A new relationship';
+        const card = cardMock();
+
+        const { wrapper } = await setup({ card });
+        wrapper.update();
+        wrapper.find('input').at(3).simulate('change', { target: { value: sideThree } });
+
+        expect(wrapper.find('input').at(3).prop('value')).toBe(sideThree);
+      });
+
+      it('should display the changed value of card side four', async () => {
+        const sideFour = 'Zombie Apocalypse';
+        const card = cardMock();
+
+        const { wrapper } = await setup({ card });
+        wrapper.update();
+        wrapper.find('input').at(4).simulate('change', { target: { value: sideFour } });
+
+        expect(wrapper.find('input').at(4).prop('value')).toBe(sideFour);
+      });
+    });
+  });
+
+  describe('Update card form', () => {
+    it('should send the filled in inputs to firebase to update card', async () => {
+      const user = { uid: '1234' };
+      const card = cardMock();
+      const spyPreventDefault = jest.fn();
+
+      const updateCard = {
+        cardTitle: card.cardTitle,
```

Comment:
So currently this test is flawed. While it passes all the subsequent card props here need to be unique. You're just writing the card mock here which gets populated anyway.